source/js/components/sticky-nav.js

```diff
@@ -24,10 +24,33 @@ var StickyNav = React.createClass({
       active: window.location.hash.replace(`#`, ``) || ``
     };
   },
+  componentDidMount: function() {
+    document.addEventListener(`scroll`, this.onScroll);
+  },
+  componentWillUnmount: function() {
+    document.removeEventListener(`scroll`, this.onScroll);
+  },
+  onScroll: function() {
```

Comment:
If it's possible we should not have things hardcoded here. We should be able to get all nav anchors by ```document.querySelectorAll(`.nav-anchor`)``` and get `id` of each element by ```.getAttribute(`id`)```

Also I think this method doesn't belong to `<StickyNav>` as we are not scrolling within the `<StickyNav>` component itself. We should move this to a page wrapper component (if there's any)